package.json

```diff
@@ -75,6 +75,8 @@
         "concurrently": "^5.3.0",
         "enzyme": "^3.11.0",
         "enzyme-adapter-react-16": "^1.15.5",
+        "lodash": "^4.17.21",
```

Comment:
maybe just add `lodash.startcase` instead?
`yarn add lodash.startcase`
and `const startCase = require('lodash.startcase');`

Reply:
Good idea :) after looking into it, that method will be deprecated, but it seems like there's a [better way](https://lodash.com/per-method-packages):
> if you import or require methods directly, e.g. const throttle = require('lodash/throttle'), only the subset of lodash code your package uses will be bundled in projects that use your package.